packages/runtime/container-runtime/src/index.ts

```diff
@@ -34,7 +31,13 @@ export {
 } from "./messageTypes.js";
 export { IBlobManagerLoadInfo } from "./blobManager.js";
 export { FluidDataStoreRegistry } from "./dataStoreRegistry.js";
-export { detectOutboundReferences } from "./dataStores.js";
+export {
+	detectOutboundReferences,
+	RuntimeHeaders,
+	DataStoresFactory,
+	AllowTombstoneRequestHeaderKey,
+	AllowInactiveRequestHeaderKey,
```

Comment:
exporting all these items seems a bit fishy. Why do we need it? Is it for tests only, or they are actually used by shipping code?

Reply:
Which ones specifically? 2 new items are being exported, are you referring to those?

Reply:
RuntimeHeaders, AllowTombstoneRequestHeaderKey, AllowInactiveRequestHeaderKey - do they need to be exposed outsiide of this package?

Reply:
yes. All of them. The last 2 are only used in tests (specifically end to end tests). They were already exported, so this PR doesn't change their visibility.